Jasmine/Common/ViewControllers/BaseGameViewControllerDelegate.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 
 /// Sets the shared functionalities that are applicable across all game view controller delegates.
 /// This protocol will be inherited by all the specialsied game view controller delegates.
-protocol BaseGameViewControllerDelegate {
+protocol BaseGameViewControllerDelegate: class {
```

Comment:
So that the delegate can be weak

http://stackoverflow.com/questions/24066304/how-can-i-make-a-weak-protocol-reference-in-pure-swift-w-o-objc